main/conceptual/ertp/purses-and-payments.md

```diff
@@ -144,48 +142,48 @@ have `payments` as arguments and effectively operate on a `payment`. The followi
 brief description and example of each `payment`-related method. For
 more detail, click the method's name to go to its entry in the [ERTP
 API Reference](/ertp/api/).
-- [`payment.getAllegedBrand()`](../api/payment.md#payment-getallegedbrand)
+- [`payment.getAllegedBrand()`](/reference/ertp-api/payment.md#payment-getallegedbrand)
   - Return the `brand` indicating the kind of digital asset this `payment` purports to be
     and which `issuer` to use with it.
     Because `payments` are not trusted, any method calls on them should be treated
     with suspicion and verified elsewhere. Any successful operation by an `issuer` on a `payment` verifies it.
 
-### Other objects' Payment-related methods:
+### Other Objects' Payment-Related Methods
```

Comment:
This seems a little odd, organizationally. But it's no worse than it was before...

Reply:
I modified the formatting of the section header to conform to the standard style I'm using for all the section headers.